src/views/DocumentationTopic.vue

```diff
@@ -9,66 +9,71 @@
 -->
 
 <template>
-  <CodeTheme class="doc-topic-view">
-    <template v-if="topicData">
-      <Nav
-        v-if="!isTargetIDE"
-        :title="topicProps.title"
-        :diffAvailability="topicProps.diffAvailability"
-        :interfaceLanguage="topicProps.interfaceLanguage"
-        :objcPath="objcPath"
-        :swiftPath="swiftPath"
-        :parentTopicIdentifiers="parentTopicIdentifiers"
-        :isSymbolDeprecated="isSymbolDeprecated"
-        :isSymbolBeta="isSymbolBeta"
-        :currentTopicTags="topicProps.tags"
-        :references="topicProps.references"
-        :isWideFormat="enableNavigator"
-        @toggle-sidenav="isSideNavOpen = !isSideNavOpen"
-      />
-      <component
-        :is="enableNavigator ? 'AdjustableSidebarWidth' : 'div'"
-        v-bind="sidebarProps"
-        v-on="sidebarListeners"
-      >
-        <template #aside="{ scrollLockID, breakpoint }">
-          <div class="doc-topic-aside">
-            <NavigatorDataProvider
-              :interface-language="topicProps.interfaceLanguage"
-              :technology="technology"
-              :api-changes-version="store.state.selectedAPIChangesVersion"
-            >
-              <template #default="slotProps">
-                <transition name="delay-hiding">
-                  <Navigator
-                    v-show="isSideNavOpen || breakpoint === BreakpointName.large"
-                    :parent-topic-identifiers="parentTopicIdentifiers"
-                    :technology="slotProps.technology || technology"
-                    :is-fetching="slotProps.isFetching"
-                    :error-fetching="slotProps.errorFetching"
-                    :api-changes="slotProps.apiChanges"
-                    :references="topicProps.references"
-                    :scrollLockID="scrollLockID"
-                    :breakpoint="breakpoint"
-                    @close="isSideNavOpen = false"
-                  />
-                </transition>
-              </template>
-            </NavigatorDataProvider>
-          </div>
-        </template>
-        <Topic
-          v-bind="topicProps"
-          :key="topicKey"
+  <div :class="quickNavigationStore.state.enableQuickNavigation ? 'modal-open' : ''">
```

Comment:
```suggestion
  <div :class="{ 'modal-open': quickNavigationStore.state.enableQuickNavigation }">
```

Reply:
Fixed in https://github.com/apple/swift-docc-render/commit/c2d8214f63cd0570d57f672a4a6a75bc83f15f70